parsing/parsetree.mli

```diff
@@ -22,7 +22,7 @@ type constant =
   (* 3 3l 3L 3n
 
      Suffixes [g-z][G-Z] are accepted by the parser.
-     Suffixes except 'l', 'L' and 'n' are rejected by the typechecker
+     Suffixes except 'l', 'L' and 'n' are rejected by the type-checker
```

Comment:
Why choose the "type-checker"  spelling rather than the "typechecker" or "type checker" ones?